test/base.test.js

```diff
@@ -305,3 +316,25 @@ test('should not set session cookie if no data in session and saveUninitialized
   t.equal(response.statusCode, 200)
   t.ok(response.headers['set-cookie'] === undefined)
 })
+
+test('should set session cookie if saveUninitialized is false and data not modified but maxAge is on', async (t) => {
```

Comment:
Actually, this is probably the wrong behavior now that I think about it...

We want the cookie to get reset if the session already has data. If it has no data, we should leave it alone.